cloudpickle/cloudpickle.py

```diff
@@ -553,6 +553,7 @@ def save_function_tuple(self, func):
             'dict': dct,
             'module': func.__module__,
             'closure_values': closure_values,
+            'name': func.__name__,
```

Comment:
this technique looks good, should we capture the rest of the things that `wraps` could update?
```
In [10]: functools.WRAPPER_ASSIGNMENTS
Out[10]: ('__module__', '__name__', '__qualname__', '__doc__', '__annotations__')
```

as it is, it looks like we drop annotations and an updated `__doc__`.